export function handleFetchError(error) {
    console.log(error);
}
